_includes/postSummary.html

```diff
@@ -23,19 +19,35 @@
     </div>
     {% endif %}
 
-    {{ post.summary | markdownify }}
+    {{ post.summary }}
```

Comment:
@WheresAlice Any reason for removing this? I'll put it back if it was removed in error